Block/Analytics.php

```diff
@@ -11,17 +11,19 @@ class Analytics extends Template
     
     public function __construct(
         Context $context,
-        \Magento\Framework\App\Action\Context            $actionContext,
-        \Metrilo\Analytics\Helper\Data                   $helper,
-        \Metrilo\Analytics\Model\Events\ProductViewEvent $productViewEvent,
-        \Metrilo\Analytics\Model\Events\PageViewEvent    $pageViewEvent,
+        \Magento\Framework\App\Action\Context             $actionContext,
+        \Metrilo\Analytics\Helper\Data                    $helper,
+        \Metrilo\Analytics\Model\Events\ProductViewEvent  $productViewEvent,
+        \Metrilo\Analytics\Model\Events\PageViewEvent     $pageViewEvent,
+        \Metrilo\Analytics\Model\Events\CategoryViewEvent $categoryViewEvent,
```

Comment:
I'm thinking:
- these are no longer models?
- if they are in the `events` folder, do we need `Event` in the name, too?

Reply:
1) Models vs Helpers in magento2 is a bit blurry topic for me and i am using this article as a guideline.
2) Since all classes are packed i don't see any reason for the classEvent suffix, tbh no idea why i put it there in first place.

Reply:
1. Fair enough
2. Then maybe remove them and we'll 🤝